docs: Add security warnings for the ONVIF document (#1045)
edgexfoundry/edgex-docs: docs_src/microservices/device/supported/device-onvif-camera/Walkthrough/custom-build.md

   cp ./cmd/res/devices/camera.yaml.example ./cmd/res/devices/camera.yaml
   ```

>       <div class='admonition warning'>
>          <p class='admonition-title'>Warning</p>
>          <p>Be careful when storing any potentially important information in cleartext on files in your computer. Potentially sensitive information in this case could include the IP address of your ONVIF camera or any custom metadata you configure.</p>
>       </div>

Reviewer: Please use the mkdocs warning here any other location where you have this embedded HTML
```suggestion
!!! Caution
    Be careful when storing any potentially important information in cleartext on files in your computer. Potentially sensitive information in this case could include the IP address of your ONVIF camera or any custom metadata you configure.
```

Reviewer: It is all about proper indentation. Mkdocs requires 4 spaces of indentation including the in-line bash block above. Once I aligned them it renders as expected.

Reviewer: ![image](https://user-images.githubusercontent.com/44779287/235742677-f9de2105-71d4-4d3b-b57e-aaa41d6dab79.png)

>       <div class='admonition warning'>
>          <p class='admonition-title'>Warning</p>
>          <p>Be careful when storing any potentially important information in cleartext on files in your computer. Potentially sensitive information in this case could include the IP address of your ONVIF camera or any custom metadata you configure.</p>
>       </div>

Reviewer: ```suggestion
    ```bash
    cp ./cmd/res/devices/camera.yaml.example ./cmd/res/devices/camera.yaml
    ```
   
    !!! warning
        Be careful when storing any potentially important information in cleartext on files in your computer. Potentially sensitive information in this case could include the IP address of your ONVIF camera or any custom metadata you configure.

```

### Configure the Device Service
1. Open the `cmd/res/configuration.yaml` file using your preferred text editor

2. Make sure `secret name` is set to match `SecretName` in `camera.yaml`. In the sample below, it is `"credentials001"`. If you have multiple cameras, make sure the secret names match.
 
3. Under `secretName`, set `username` and `password` to your camera credentials. If you have multiple cameras copy the `Writable.InsecureSecrets` section and edit to include the new information.

    !!! warning
        Be careful when storing any potentially important information in cleartext on files in your computer. In this case, the credentials for the camera(s) are stored in cleartext in the `configuration.yaml` file on your system.
        `InsecureSecrets` is for non-production use only.

    !!! example - "Sample: Snippet from configuration.yaml"
        ```yaml
        Writable:
        LogLevel: INFO
        InsecureSecrets:
           credentials001:
              SecretName: credentials001
              SecretData:
              username: <Credentials 1 username>
              password: <Credentials 1 password>

>               mode: usernametoken   # assign "digest" | "usernametoken" | "both" | "none"

Reviewer 2: I think the indents are off. Shouldn't these fields be nested under SecretData?

Reviewer: @ajcasagrande is correct

>               SecretData:
>               username: <Credentials 1 username>
>               password: <Credentials 1 password>
>               mode: usernametoken   # assign "digest" | "usernametoken" | "both" | "none"

Reviewer: ```suggestion
              SecretData:
                 username: <Credentials 1 username>
                 password: <Credentials 1 password>
                mode: usernametoken   # assign "digest" | "usernametoken" | "both" | "none"
```

Reviewer 2: agree. please make this change. (even if the mistake existed before).

1. Make sure `secret name` is set to match `SecretName` in `camera.yaml`. In the sample below, it is `"credentials001"`. If you have multiple cameras, make sure the secret names match.

      
1. Under `secretName`, set `username` and `password` to your camera credentials. If you have multiple cameras copy the `Writable.InsecureSecrets` section and edit to include the new information.

      <div class='admonition warning'>
         <p class='admonition-title'>Warning</p>

>          <p>Be careful when storing any potentially important information in cleartext on files in your computer. In this case, the credentials for the camera(s) are stored in cleartext in the `configuration.yaml` file on your system. </p>

Reviewer: I would add something about `InsecureSecrets` is for non-production use since the credentials are stored in plain text

      </div>

>       ```yaml
>       Writable:
>       LogLevel: INFO
>       InsecureSecrets:
>          credentials001:
>             SecretName: credentials001
>             SecretData:
>             username: <Credentials 1 username>
>             password: <Credentials 1 password>
>             mode: usernametoken   # assign "digest" | "usernametoken" | "both" | "none"
>          credentials002:
>             SecretName: credentials002
>             SecretData:
>             username: <Credentials 2 username>
>             password: <Credentials 2 password>
>             mode: usernametoken    # assign "digest" | "usernametoken" | "both" | "none"
>       ```

Reviewer: Please use mkdocs example

```suggestion
!! example - "Sample: Snippet from configuration.yaml"
     ```yaml
      Writable:
      LogLevel: INFO
      InsecureSecrets:
         credentials001:
            SecretName: credentials001
            SecretData:
            username: <Credentials 1 username>
            password: <Credentials 1 password>
            mode: usernametoken   # assign "digest" | "usernametoken" | "both" | "none"
         credentials002:
            SecretName: credentials002
            SecretData:
            username: <Credentials 2 username>
            password: <Credentials 2 password>
            mode: usernametoken    # assign "digest" | "usernametoken" | "both" | "none"
    ```
```

Reviewer: This one still needs to be addressed.

>       <p align="left">
>          <i>Sample: Snippet from configuration.yaml</i>
>       </p>

Reviewer: ```suggestion
```
